eva-accession-release/src/test/resources/test-data/dbsnp-mirror-schema.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE dbsnp_variant_load_nohgvslink_765483204148e0ba5005211d6ecd3331 (
```

Comment:
Could we try to use the same test files that are already in the core module?

Reply:
We are now using the test files from the core module when possible (data for chicken). In the case of this file it is not in the core because it is for vervet monkey and this species is only used in the release module.